.github/workflows/commit_epm.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     strategy:
       matrix:
         sgx: [[self-hosted, SGX2]]
-        tag: [ubuntu18.04, centos8.2]
+        tag: [ubuntu18.04, centos8.2, alinux2]
     steps:
```

Comment:
Please remove all *deb *rpm and other files on the host which are generated in the test process.